### ISSUE TITLE
Icon request: icon-geometric-solids 

### ISSUE THREAD

**Issue by bjtb** [FortAwesome/Font-Awesome#13326]
TL;DR: We need an icon for a general object in our app. E.g. circle+square+triangle in one icon, or any other icon that may represent a general, physical object.

![image](https://user-images.githubusercontent.com/40008894/41419171-a584fa76-6ff1-11e8-879c-5b1fefe7cddd.png)

Detailed description:
In our app, we have two concepts that are similar: Products and Objects. An object can be anything that may need maintenance or repair (a car, power pole, machinery, etc.) while products are materials/supplies/stock/parts used to repair objects, or other items that may be needed for an assignment and should be charged to the customer. As our app has users from a vast range of industries, both these icons should be quite general. We use a box to represent products, but we need a different and preferably just as general icon for objects.

**Comment by bjtb** [comment]
@tagliala How on earth are other peope going to find and vote for "icon-geometric-solids"?

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it already exists in Font Awesome Pro! [Take a look at it here.](https://fontawesome.com/icons/shapes?f=classic&s=solid)
